.github/workflows/docker-release-artifact-registry.yml

```diff
@@ -0,0 +1,161 @@
+name: Docker Release to GCS Artifact Registry
```

Comment:
why we needed to duplicate the release pipeline? can we just do it in the last step of docker-release2?

Reply:
we can do it in docker-release2. Moving it.